src/tdastro/graph_state.py

```diff
@@ -60,6 +60,17 @@ def __init__(self, num_samples=1):
     def __len__(self):
         return self.num_parameters
 
+    def __contains__(self, key):
+        if key in self.states:
+            return True
+        elif self._NAME_SEPARATOR in key:
+            tokens = key.split(self._NAME_SEPARATOR)
+            if len(tokens) != 2:
+                raise KeyError(f"Invalid GraphState key: {key}")
+            return True
```

Comment:
Do we still want to check if self.states contains the tokens?

Reply:
Nice catch. This is a bug. I'm pushing a change to fix it (and corresponding tests).